pkgs/development/roswell/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, curl, autoreconfHook, makeWrapper, sbcl}:
+
+stdenv.mkDerivation rec {
+  pname = "roswell";
+  version = "21.01.14.108";
+  src = fetchFromGitHub {
+    owner = "roswell";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1hj9q3ig7naky3pb3jkl9yjc9xkg0k7js3glxicv0aqffx9hkp3p";
+  };
+
+  configurePhase = ''
+    runHook preConfigure
+    sh bootstrap && ./configure --prefix $out
+    runHook postConfigure
+  '';
```

Comment:
```suggestion
  preConfigure = ''
    ./bootstrap
  '';
  
  configureFlags = [ "--prefix=$out" ];
```

Reply:
that breaks it bc. it tries to init setuptools "the wrong way"

Reply:
We should use the configure phase by nix instead of overwriting everything.

Reply:
have you tried building the derivation with that change?